test/functional/Java8andUp/src/org/openj9/resources/reflect/A.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2018 IBM Corp. and others
```

Comment:
For a new file, the start year is `2018` as well.
Not sure if you are an IBM employee, I think you are free to choose a copyright for your own institution.
@DanHeidinga @pshipton please comment if this is not accurate.